Enhance the Zowe CLI sample plug-in to provide an SCRT test tool (#147)
zowe/zowe-cli-sample-plugin: README.md

* Make use of Zowe CLI infrastructure (profiles and programmatic APIs).
* Participate in the Zowe CLI community space.

## Prerequisites

> ### Install Zowe CLI before you work with the Zowe CLI sample plug-in.

Reviewer: ```suggestion
### Install Zowe CLI before you work with the Zowe CLI sample plug-in
```

> See [Installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli#installing-zowe-cli-and-zowe-cli-plug-ins-from-a-local-package)

Reviewer: ```suggestion
See [Installing Zowe CLI](https://docs.zowe.org/stable/user-guide/cli-installcli#installing-zowe-cli-and-zowe-cli-plug-ins-from-a-local-package).
```

### Obtain the zowe-cli-sample-plugin

> Most of this plugin is merely a sample for you to program your own plug-in. For that purpose, you should clone the repository and build the sample plug-in from source. However, one feature of the plug-in can be used as a tool to test the transmission of SCRT data to a targeted REST service. To simply use the plugin for testing purposes, you can download a published package of the zowe-cli-sample-plugin.

Reviewer: ```suggestion
Most of this plug-in is merely a sample for you to program your own plug-in. For that purpose, you should clone the repository and build the sample plug-in from source. However, one feature of the plug-in can be used as a tool to test the transmission of SCRT data to a targeted REST service. To simply use the plug-in for testing purposes, you can download a published package of the zowe-cli-sample-plugin.
```

#### Download a published version of the zowe-cli-sample-plugin

> 1. Download the sample plug-in from the Zowe JFrog registry.

Reviewer: ```suggestion
1. Download the sample plug-in from the Zowe JFrog registry:
```

   
    `npm pack @zowe/sample-plugin-for-zowe-cli@latest -registry https://zowe.jfrog.io/zowe/api/npm/npm-local-release/`
   

>     If your site restricts access to zowe.jfrog.io, you may need to use a corporate internal registry.

Reviewer: ```suggestion
    If your site restricts access to zowe.jfrog.io, you might need to use a corporate internal registry.
```

> 2. Install the sample plug-in using your previously installed Zowe CLI and the sample plug-in package that you downloaded. In the example below we used the 4.1.0 version of the sample plug-in.

Reviewer: ```suggestion
2. Install the sample plug-in using your previously installed Zowe CLI and the sample plug-in package that you downloaded. In the example below we used the 4.1.0 version of the sample plug-in:
```

   
    `zowe plugins install zowe-sample-plugin-for-zowe-cli-4.1.0.tgz`

> #### Customize the supplied sample config file

Reviewer: ```suggestion
#### Customize the supplied sample configuration file
```

> To quickly use a config file appropriate for the sample plugin, view the Readme file and customize the sample Zowe config file located in the folder [customize_your_config_file](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/docs/tutorials/customize_your_config_file).

Reviewer: ```suggestion
To quickly use a configuration file appropriate for the sample plug-in, view the Readme file and customize the sample Zowe config file located in the folder [customize_your_config_file](https://github.com/zowe/zowe-cli-sample-plugin/blob/master/docs/tutorials/customize_your_config_file).
```

**Note:** For an advanced example on how to create a plug-in that serves as a Credential Manager, see [Zowe CLI secrets for Kubernetes](https://github.com/zowe/zowe-cli-secrets-for-kubernetes/)

> The purpose and use of this plugin's  `send check-status` command is documented in [Using the 'send check-status' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-check-status/SendZosmfCheckStatusCommand.md)

Reviewer: ```suggestion
The purpose and use of this plug-in's  `send check-status` command is documented in [Using the 'send check-status' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-check-status/SendZosmfCheckStatusCommand.md).
```

> The purpose and use of this plugin's  `send scrt` command is documented in [Using the 'send scrt' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-scrt/SendScrtCommand.md)

Reviewer: ```suggestion
The purpose and use of this plug-in's  `send scrt` command is documented in [Using the 'send scrt' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-scrt/SendScrtCommand.md).
```

**Note:** For an advanced example on how to create a plug-in that serves as a Credential Manager, see [Zowe CLI secrets for Kubernetes](https://github.com/zowe/zowe-cli-secrets-for-kubernetes/)

The purpose and use of this plug-in's  `send check-status` command is documented in [Using the 'send check-status' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-check-status/SendZosmfCheckStatusCommand.md).

> The purpose and use of this plug-in's  `send scrt` command is documented in [Using the 'send scrt' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-scrt/SendScrtCommand.md).

Reviewer 2: (nit) extra space before inline code blocks for command names

```suggestion
The purpose and use of this plug-in's `send check-status` command is documented in [Using the 'send check-status' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-check-status/SendZosmfCheckStatusCommand.md).

The purpose and use of this plug-in's `send scrt` command is documented in [Using the 'send scrt' command](https://github.com/zowe/zowe-cli-sample-plugin/docs/tutorials/send-scrt/SendScrtCommand.md).
```
